deployment/cdk/opensearch-service-migration/lib/fetch-migration-stack.ts

```diff
@@ -67,8 +67,8 @@ export class FetchMigrationStack extends Stack {
 
         // Create DP pipeline config from template file
         let dpPipelineData: string = readFileSync(props.dpPipelineTemplatePath, 'utf8');
+        // Replace only source cluster host - target host will be overridden by inline env var
         dpPipelineData = dpPipelineData.replace("<SOURCE_CLUSTER_HOST>", props.sourceEndpoint);
-        dpPipelineData = dpPipelineData.replace("<TARGET_CLUSTER_HOST>", targetClusterEndpoint);
```

Comment:
This part is a bit confusing for me. We will no longer replace <TARGET_CLUSTER_HOST> in the yaml file but it still exists. Do we replace it in the Python somewhere with the environment variable (I looked but may have missed this)?

Reply:
On a different note, we should add a small note in the README (or if we can do a comment in the pipeline yaml even better) for setting up FetchMigration that a user does not need to fill in the `SOURCE_CLUSTER_HOST` or `TARGET_CLUSTER_HOST` values in the yaml but that they will be autofilled by us.

Reply:
> This part is a bit confusing for me. We will no longer replace <TARGET_CLUSTER_HOST> in the yaml file but it still exists. Do we replace it in the Python somewhere with the environment variable (I looked but may have missed this)?

Yes, that's what the [write_inline_target_host function](https://github.com/kartg/opensearch-migrations/blob/1333db5de17876e2767667c9e5f7034b18adb336/FetchMigration/python/fetch_orchestrator.py#L63) does. `<TARGET_CLUSTER_HOST>` no longer needs to be specified because we're not performing a simple string substitution anymore - rather, the Python function parses the Data Prepper pipeline definition structurally and replaces the `host` value of the appropriate `sink` plugin

Reply:
> On a different note, we should add a small note in the README (or if we can do a comment in the pipeline yaml even better) for setting up FetchMigration that a user does not need to fill in the SOURCE_CLUSTER_HOST or TARGET_CLUSTER_HOST values in the yaml but that they will be autofilled by us.

Included in a parallel PR 😄

https://github.com/opensearch-project/opensearch-migrations/pull/390/files#diff-4e4fd6e3cb3114763fe0616b8fb3d9af7dacae3d58faee516575d30054ac599d

Reply:
Great thanks!